microservices/in-app-notification-dispatcher/dev/values.yaml

```diff
@@ -14,7 +14,7 @@ configmap:
 deployment:
   flywayInitContainer:
     create: true
-    migrationsConfigmap: "in-app-notification-db-migrations"
+    migrationsConfigmap: "flyway-in-app-notification"
```

Comment:
I'd left the old configmap name here, this is the new one, same as is used for the in-app-notification-manager (we decided both services would run those migrations):
https://github.com/pagopa/interop-core-deployment/blob/e24cd5e92b2a602d9e1be56bcc75c6d423921909/microservices/in-app-notification-manager/dev/values.yaml#L17